_data/speakers.yml

```diff
@@ -471,17 +490,20 @@
   image: profile_pgr.jpg
   thumbnail: profile_pgr.jpg
   alt: Pedro
-  topic:
-  proposal_summary:
+  topic: Sequential Monte Carlo: Introduction and diagnostics
+  proposal_summary: Sequential Monte Carlo (also known as particle filters) were initially confined to the so-called "filtering problem" (the sequential analysis of state-space models), but they popularity has been growing in other areas including non-sequential tasks like estimating a single posterior distribution, i.e. the typical task performed by pm.sample(). For such cases SMC samplers has been presented as alternatives to MCMC methods. Some advantages often listed in the literature are: simple way of estimating the marginal likelihood (generally for the purpose of model choice), easier parallelization, easier design of adaptative methods, and ability to deal with multi-modal posteriors.
+  The literature usually fails to acknowledge that SMC methods are a less mature alternative when compared to Hamiltonian Monte Carlo methods, such as the nuts sampler in PyMC3, Stan, etc. SMC methods are also the base for the most advanced Approximate Bayesian Computation (ABC) methods, which are also increasingly used to solve complex problems in fields like polulation genetics, systems biology, astronomy among others.
+  The literature on SMC and SMC-ABC methods can be puzzling for newcomers as the same methods has more than one name or sometimes different names correspond to the same method or concept. In this talk we will provide a brief introduction to SMC methods specially focused on clarifying the key concepts and terminology necessary to use SMC methods in practice, and then we will focus on how to diagnose SMC, including results from preliminary experiments.
   discourse_link:
   youtube_link:
   description: In the year 2014 I completed my Bs. in Molecular Biology at the National University of San Luis, Argentina
     and in 2020 I finished my PhD in the Instute of Applied Mathematics (IMASL) while working within the Structural Bioinformatics Group (BIOS).
     My PhD thesis was centered around the use of a statiscal mechanics model to simulate biologically relevant systems of peptide-lipid interactions.
     Currently I'm doing my postdoc alongside Dr. Osvaldo Martin on probabilistic modeling and Sequential Monte Carlo.
   presentation_type: Talk
-  time_zone:
+  time_zone: Americas
   track: Advanced
+  keywork: Pedrez
```

Comment:
Sequential monte carlo talk has 2 speakers, both should have the same keyword.

Reply:
Changed the keywork